charts-external/openpension/templates/service.yaml

```diff
@@ -0,0 +1,12 @@
+{{ if .Values.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: openpension
```

Comment:
need to modify name to e.g. openpension-client and the selector labels accordingly